docs/docs-beta/docs/guides/external-systems/apis.md

```diff
@@ -1,4 +1,53 @@
 ---
 title: Connecting to APIs
 sidebar_position: 20
----
\ No newline at end of file
+---
+
+This guide describes how to connect to and interact with APIs in dagster.
+
+
+## What you'll learn
+
+- How to write a dagster Resource to connect to an API
```

Comment:
```suggestion
- How to connect to an API using a Dagster resource
```

Feels nitpicky, but I think switching around the what and how here is more user-centric